docs/src/index.md

```diff
@@ -26,6 +26,29 @@ in Julia.
     Your changes will be vetted by developers before becoming permanent, so don't
     worry about whether you might say something wrong.
 
-```@contents
-Pages = ["install.md", "quickstart.md", "arrays_colors.md", "conversions_views.md", "indexing.md", "imageaxes.md", "imagefiltering.md", "imagemetadata.md", "imagesegmentation.md", "function_reference.md", "api_comparison.md"]
+This documentation is a collection of two parts: a top-level documentation that guides you into the
+JuliaImages ecosystem, and a list of more specific documentation of sub-packages that gives you more
+information of each image processing subfields.
+
+The "Tutorials" part contains a list of tutorials that help you gain better understanding of the JuliaImages
+ecosystems. The ["Packages"](@ref page_packages_index) part contains all related sub-packages in JuliaImages,
+users can get more information on the specific image processing subfields by reading them. The "Demos"
+part gives you a task-by-task introduction of how you could do your daily work with JuliaImages.
+The "References" part is a collection of function references, it's a dictionary of functions in JuliaImages
+and the recommended way to use it is by the searching function of your browser `Ctrl-F`/`Cmd-F`.
+
+This documentation is generated with the following environment setup. While reading the documentation,
+if you encounter any errors or if the outputs in your local machine differ from the documentation,
+you could first check the Julia and package versions you're using. If the error or inconsistency still exists,
+please [file an issue](https://github.com/JuliaImages/juliaimages.github.io/issues/new) for that; it
+helps us improve the documentation.
+
+```@setup versions
+using InteractiveUtils
+```
+```@repl versions
+using Pkg, Dates
+today()
+versioninfo()
+Pkg.status()
```

Comment:
Is this really what we need for the top page? I think the top page should be a "welcome" page, as @timholy implies in https://github.com/JuliaImages/juliaimages.github.io/pull/125#discussion_r412076360.

Reply:
Another option is using

    ```@raw html
    <details>
      <summary>Package Status</summary>
    ```
and

    ```@raw html
    </details>
    ```
I don't really recommend it, though,

Reply:
I believe it is necessary to share the build information somewhere in the documentation. I've now moved this to the troubleshooting section in `install.md`.

> we can find the date and the version from the top-right icon.

Yes but I feel putting them together more convenient to get relevant information.